Library/Homebrew/test/Gemfile

```diff
@@ -3,6 +3,7 @@ source "https://rubygems.org"
 gem "mocha", "~> 1.1"
 gem "minitest", "~> 5.3"
 gem "rake", "~> 10.3"
+gem "test-unit", "~> 3.2"
```

Comment:
We should get a merge conflict when upstream bumps the other versions, at which point we can bump `test-unit`.